### DISCUSSION TITLE
rocketmq5.3.0使用acl2.0部署，启动失败

### DISCUSSION THREAD

**Discussion by jaylu163** [apache/rocketmq#9005] [Q&A]
问题1： 使用acl 1.0开启认证(aclEnable=true)，没有开启acl 2.0的配置。启动失败。

报错如下：
![image](https://github.com/user-attachments/assets/939fc940-99c2-483c-9192-732e03faecd9)

问题2： 使用acl 2.0开启认证后，proxy启动失败
proxy启动的日志如图：
![image](https://github.com/user-attachments/assets/8591f2d9-5804-4ccb-9e0b-6bf8012e3a6f)

broker的配置：
![image](https://github.com/user-attachments/assets/8edca5ee-1ddd-4515-b0d4-a7366ba2b94e)
proxy的配置：
![image](https://github.com/user-attachments/assets/b502d63b-375d-45cd-8831-9b4c7c20a0ea)

求大佬帮看看是哪块问题呢

**Comment by hexueyuan** [comment]
从第一个截图可以看到broker接收到的请求里没找到用户名，可以按错误栈跟一下proxy哪里配置的不对。

**Reply by jaylu163** [reply]
找到了这个问题，太坑了，感觉是官方的bug

配置开启了这个后，不会增加 DefaultHeartBeatSyncerTopic 这个topic,
